### ISSUE TITLE
Gradio UI app to generate audio from text

### ISSUE THREAD

**Issue by huynhnhathao** [gitmylo/bark-voice-cloning-HuBERT-quantizer#58]
For anyone who is interested, I have implemented a Gradio UI app of BARK that is ready to run on your local computer, plus most of the code in this codebase is reimplemented to be more readable
I also trained a HuBERT model to predict the semantic tokens from audio from a more than 4700 generated ~14s examples of audio-semantic dataset, the validation accuracy of my model is 83% on more than 15k tokens, one can get a feeling of cloning a voice but it is not perfect to impersonate anyone, plus you can do all that in the UI, touching no code

I also provided the script to train your finetune HuBERT model and to create new wave-semantic dataset

https://github.com/huynhnhathao/bark_text_to_audio

![Image](https://github.com/user-attachments/assets/09e16351-b0f2-4d14-8b96-3be0b7f4c2a1)
